ush/python/pygfs/task/marine_letkf.py

```diff
@@ -43,6 +71,53 @@ def initialize(self):
 
         logger.info("initialize")
 
+        # make directories and stage ensemble background files
+        letkf_stage_list = parse_j2yaml(self.task_config.MARINE_LETKF_STAGE_YAML_TMPL, self.task_config)
+        FileHandler(letkf_stage_list).sync()
+
+        # TODO(AFE): probably needs to be jinjafied
+        obs_list = YAMLFile(self.task_config.OBS_YAML)
+
+        # get the list of observations
+        CDATE = datetime_to_YMDH(self.runtime_config.current_cycle)
+        obs_files = []
+        for ob in obs_list['observers']:
+            obs_name = ob['obs space']['name'].lower()
+            obs_filename = f"{self.task_config.RUN}.t{self.task_config.cyc}z.{obs_name}.{CDATE}.nc4"
+            obs_files.append((obs_filename, ob))
+
+        obs_files_to_copy = []
+        obs_to_use = []
+        # copy obs from COM_OBS to DATA/obs
+        for obs_file, ob in obs_files:
+            logger.info(f"******* {obs_file}")
```

Comment:
Do we need all this logging here? FileHandler will tell you where it is copying from/to

Reply:
removed/changed